app/models/user.rb

```diff
@@ -17,10 +17,15 @@ class User < ApplicationRecord
   has_one :supervisor_volunteer, foreign_key: "volunteer_id"
   has_one :supervisor, through: :supervisor_volunteer
 
-  def self.volunteers_with_no_supervisor(org)
-    # TODO make this a scope
-    Volunteer.active.includes(:supervisor_volunteer).where(casa_org_id: org.id).where(supervisor_volunteers: {id: nil})
-  end
+  scope :volunteers_with_no_supervisor, lambda { |org|
```

Comment:
Wow wow wow wow! This looks great. I need to give this a deeper review, it looks great so far and I will dig in more tonight :)